workalendar/tests/__init__.py

```diff
@@ -13,11 +13,12 @@
 
 class CoreCalendarTest(TestCase):
     cal_class = Calendar
+    kwargs = {}
```

Comment:
I had to find a way to pass the `region` to the test class. I'm not sure if there's a better way to do that. Here I made it generic to pass such variables, which might also be useful for others?